web/backend_api_builder.py

```diff
@@ -22,6 +22,10 @@ def member_id(self, member_id):
         """
         return self.members + '/{}'.format(member_id)
 
+    @property
+    def member_search(self):
+        return self.base + 'members/search'
```

Comment:
you can just do `self.members` + '/search'

Reply:
ok